test/powershell/engine/Module/TestModuleManifest.ps1

```diff
@@ -0,0 +1,13 @@
+Describe "Test-ModuleManifest tests" -tags "CI" {
+
+    It "module manifest containing paths with backslashes or forwardslashes are resolved correctly" {
+
+        New-Item -ItemType Directory -Path testdrive:/module
+        New-Item -ItemType Directory -Path testdrive:/module/foo
+        New-Item -ItemType Directory -Path testdrive:/module/bar
+        "" > testdrive:/module/foo/bar.psm1
+        "" > testdrive:/module/bar/foo.psm1
+        New-ModuleManifest -NestedModules foo\bar.psm1,bar/foo.psm1 -RootModule foo\bar.psm1 -RequiredAssemblies foo/bar.psm1,bar\foo.psm1 -Path testdrive:/module/test.psd1
+        Test-ModuleManifest -Path testdrive:/module/test.psd1 -ErrorAction Stop | Should BeOfType System.Management.Automation.PSModuleInfo
```

Comment:
Do you want some additional validation on the paths, maybe a `Test-Path` on the `NestedModules`, `RootModule`, and `RequiredAssemblies`.

There are other module properties that specify files, maybe you should be testing those too - `TypesToProcess`, `FormatsToProcess`, `ScriptsToProcess`, maybe `FileList` and `ModuleList`?

I wonder if it's a bad idea to specify `RequiredAssemblies` without a valid assembly - if that's not already causing problems, I can see it causing problems in the future.